nnedi3/nnedi3.h

```diff
@@ -27,6 +27,19 @@
 #include "./avisynth.h"
 #include "./PlanarFrame.h"
 #include "./ThreadPoolInterface.h"
+#ifndef _MSC_VER
+#define _strnicmp strncasecmp
+#endif
+
+#include <stdlib.h>
+
+using namespace std;
+ 
+#ifndef _MSC_VER
+#define _aligned_malloc(size, alignment) aligned_alloc(size, alignment)
+// Note: This is safe on Unix. see `man 3 aligned_alloc`
+#define _aligned_free(m) free(m)
```

Comment:
@jpsdr You might be right about this on Windows but using free on aligned memory is fine on Unix.